tools/Customization/DevHome.Customization/Views/DeveloperFileExplorerPage.xaml.cs

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System.Collections.ObjectModel;
+using DevHome.Common.Extensions;
+using DevHome.Common.Services;
+using DevHome.Customization.ViewModels;
+using DevHome.Settings.Models;
+using Microsoft.UI.Xaml;
+using Microsoft.UI.Xaml.Controls;
+
+namespace DevHome.Customization.Views;
+
+public sealed partial class DeveloperFileExplorerPage : Page
+{
+    public DeveloperFileExplorerViewModel ViewModel
+    {
+        get;
+    }
+
+    public ObservableCollection<Breadcrumb> Breadcrumbs
```

Comment:
I would nudge you to move `Breadcrumbs` to the `ViewModel`

Reply:
@dhoehna https://github.com/microsoft/devhome/pull/2383#discussion_r1520332745

Reply:
Decided to just pull the Breadcrumbs for now, we can revisit a more generic solution that would apply here after this goes in.